packages/cache-core/src/cacheHandler.ts

```diff
@@ -86,9 +93,23 @@ export class Cache implements CacheHandler {
   }
 
   getCurrentDeviceType() {
-    return Cache.enableDeviceSplit
-      ? parser(this.nextOptions._requestHeaders['user-agent'] as string)?.device?.type ?? ''
-      : ''
+    if (!Cache.enableDeviceSplit) return ''
+    const headers = this.nextOptions._requestHeaders
+
+    if (headers['user-agent'] === 'Amazon CloudFront') {
+      if (headers[HEADER_DEVICE_TYPE.Desktop] === 'true') {
+        return ''
```

Comment:
this is inconsistent in comparison to user agent parser logic, because it is going to return desktop

Reply:
Do you mean this one? https://www.npmjs.com/package/ua-parser-js
I checked and desktop is not a possible device type so let's not add any types for desktop.